x-pack/plugins/siem/public/resolver/view/process_event_dot.tsx

```diff
@@ -251,8 +256,33 @@ const ProcessEventDotComponents = React.memo(
      * position to accomodate for that. This aligns the logical center of the process node
      * with the correct position on the map.
      */
-    const processNodeViewXOffset = -0.172413 * logicalProcessNodeViewWidth * magFactorX;
-    const processNodeViewYOffset = -0.73684 * logicalProcessNodeViewHeight * magFactorX;
+
+    const isZoomedOut = magFactorX < 0.8;
+    const isShowingDescriptionText = magFactorX >= 0.55;
+
+    const nodeXOffsetValue = isZoomedOut
+      ? -0.143413 + (-magFactorX * (1 - magFactorX)) / 10
+      : -0.147413;
+    const nodeYOffsetValue = isZoomedOut
+      ? -0.53684 + (-magFactorX * 0.2 * (1 - magFactorX)) / magFactorX
+      : -0.53684;
+
+    const actionsBaseYOffsetPct = 5;
```

Comment:
can you explain this?

Reply:
I can rename it and add a description. It sets the top % offset for the process buttons to keep them aligned as the resolver scales down